packages/nx/src/nx-cloud/utilities/onboarding.ts

```diff
@@ -0,0 +1,34 @@
+import { readNxJson } from '../../devkit-exports';
+import type { Tree } from '../../generators/tree';
+import { NxCloudOnBoardingStatus } from '../models/onboarding-status';
+import { isWorkspaceClaimed } from './is-workspace-claimed';
+import { createNxCloudOnboardingURL } from './url-shorten';
+import { getRunnerOptions } from '../../tasks-runner/run-command';
+
+export async function getNxCloudOnBoardingStatus(
+  tree: Tree,
+  token?: string
+): Promise<NxCloudOnBoardingStatus> {
+  token = token || readNxCloudToken(tree);
+  if (!token) {
+    return 'not-configured';
+  }
+  return (await isWorkspaceClaimed(token)) ? 'claimed' : 'unclaimed';
+}
+
+export async function getNxCloudAppOnBoardingUrl(token: string) {
+  if (!token) {
+    return null;
+  }
+  const onboardingUrl = await createNxCloudOnboardingURL(
+    'create-nx-project',
```

Comment:
Let's make this `nx-welcome-app`.

Can we make this function name something more clear like `createNxCloudOnboardingURLForWelcomeApp`?

It's currently named basically the same as the `createNxCloudOnboardingURL`